packages/workspace/src/migrations/update-13-2-0/set-parallel-default.ts

```diff
@@ -4,18 +4,19 @@ import {
   Tree,
   updateWorkspaceConfiguration,
 } from '@nrwl/devkit';
-import { output } from '../../utilities/output';
 
 export async function setParallelDefault(host: Tree) {
   const config = readWorkspaceConfiguration(host);
-  if (config.tasksRunnerOptions['default'].options.parallel) {
-    config.tasksRunnerOptions['default'].options.parallel =
-      config.tasksRunnerOptions['default'].options.maxParallel || 3;
-    delete config.tasksRunnerOptions['default'].options.maxParallel;
-  } else {
-    config.tasksRunnerOptions['default'].options.parallel = 1;
+  if (config.tasksRunnerOptions && config.tasksRunnerOptions['default']) {
+    if (config.tasksRunnerOptions['default'].options.parallel) {
```

Comment:
Maybe we can pull a lot of this out into a variable to keep repetition down. Maybe between lines 9-10 add something like this:
```typescript
const defaultTaskRunnerOptions = config.tasksRunnerOptions?.['default'].options
```

Then that could be used in the if conditions and during assignment.

Reply:
Good suggestion, updated now 👍